doc/nrf/links.txt

```diff
@@ -1045,9 +1045,6 @@
 .. _`nRF Cloud REST API ProvisionDevices`: https://api.nrfcloud.com/v1#tag/IP-Devices/operation/ProvisionDevices
 .. _`nRF Cloud RegisterPublicKeys Endpoint`: https://api.nrfcloud.com/v1/#operation/RegisterPublicKeys
 
-.. _`nRF Cloud Portal Provision Devices`: https://nrfcloud.com/#/provision-devices
```

Comment:
do we not want to use this link?
[https://nrfcloud.com/#/add-device/bulk](https://nrfcloud.com/#/add-device/bulk)

```
.. _`nRF Cloud Portal Onboard Devices`: https://nrfcloud.com/#/add-device/bulk
```

Reply:
actually, replace line 1017:
```
.. _`Provision Devices`: http://nrfcloud.com/#/provision-devices
```
with
```
.. _`Onboard Devices`: https://nrfcloud.com/#/add-device/bulk
```

Reply:
We do not want to use these links because they would take you only to the portal front page if you are not logged into the portal.

Reply:
Actually, is the portal smart enough to take the user to the correct page after logging in?

Reply:
Let me try.

Reply:
Seems to work, when I am logged in.